src/ck-ldb/LBManager.C

```diff
@@ -445,7 +474,8 @@ void LBManager::initnodeFn()
 void LBManager::callAt()
 {
   localBarrier.CallReceivers();
-  if (nloadbalancers > 0) loadbalancers[0]->InvokeLB();
+  //TODO: Add support for sequencing multiple LBs
+  if (loadbalancers.size() > 0) loadbalancers[0]->InvokeLB();
```

Comment:
Should we fix this in this PR to support calling multiple LBs or make a note in documentation to say that using multiple +balancer option is temporarily not supported?

Reply:
We should add support for multiple LBs, but I think that can go into a subsequent PR, if that's alright; I don't want to delay this one anymore.